src/windshaft-integration/model-updater.js

```diff
@@ -40,7 +40,8 @@ ModelUpdater.prototype.updateModels = function (windshaftMap, sourceId, forceFet
 
 ModelUpdater.prototype._updateLayerGroupModel = function (windshaftMap) {
   var urls = {
-    tiles: this._calculateTileURLTemplatesForCartoDBLayers(windshaftMap),
+    tiles: this._generateTileURLTemplate(windshaftMap),
+    subdomains: this._getTileSubdomains(windshaftMap),
```

Comment:
`subdomains` is not a url so perhaps it makes sense to set it as a separate attribute IMO.